mne/preprocessing/ica.py

```diff
@@ -2017,6 +2021,24 @@ def save(self, fname, *, overwrite=False, verbose=None):
             _write_ica(fid, self)
         return self
 
+    @property
+    def explained_variance_(self):
```

Comment:
sklearn uses `explained_variance_ratio_` attribute for this

Reply:
Great idea and avoids confusion with `pca_explained_variance_`, which is **not** a ratio